src/libtiled/orthogonalrenderer.cpp

```diff
@@ -311,7 +311,7 @@ void OrthogonalRenderer::drawTileLayer(QPainter *painter,
     const QTransform savedTransform = painter->transform();
     painter->translate(layerPos);
 
-    CellRenderer renderer(painter, this);
+    CellRenderer renderer(painter, this, layer->tintColor(), CellRenderer::CellType::OrthogonalCells);
```

Comment:
No need to add `CellRenderer::CellType::OrthogonalCells` here since that the default value.